index.html

```diff
@@ -421,4 +438,35 @@ <h3 class="about">About me</h3>
 
     <div class="break1"></div>
   </footer>
+
+  <article id="card" class="card-container invisible">
+    <div class="card">
+      <i id="card-cross" class="fas fa-times card-cross" onclick="toggle()"></i>
+      <h3 id="card-title">See Project header</h3>
+      <ul id="tech-list" class="tech">
+      </ul>
+      <div class="popupFooter">
+        <img src="images/card1" class="card-image" alt="project screenshot">
+        <iframe src="" frameborder="0" class="card-iframe invisible"></iframe>
+        
+
+        <div class="functionality">
+          <p id="card-description">
+          </p>
+          <div class="button-container">
+            <button type="button" id="card-button-live" class="card-button">
+                See live
```

Comment:
- [ ] Since you are asked to create everything dynamically using JS, please do not write again HTML code for it, what you can do in this section is just create an empty container as this one is an `article` tag, okay, just create it empty `<article> </article>` and the whole content should be generated through javascript, this will be much cleaner and efficient.